profiles/common/modules/features/nexteuropa_metatags/nexteuropa_metatags.module

```diff
@@ -10,6 +10,13 @@ include_once 'nexteuropa_metatags.tokens.inc';
  * Implements hook_metatag_info().
  */
 function nexteuropa_metatags_metatag_info() {
+  $info['group']['tags'] = array(
```

Comment:
@voidtek She will be back on Tuesday. If you think that this is unrelated changes, please, update the PR so we do not lose too much time.